internal/commands/main.go

```diff
@@ -162,22 +169,14 @@ func (c Commands) FindArticle(substr string) (item rss.Item, err error) {
 		return rss.Item{}, fmt.Errorf("commands.FindArticle: %w", err)
 	}
 
-	regex, err := regexp.Compile(strings.ToLower(substr))
-	if err != nil {
-		return rss.Item{}, fmt.Errorf("commands.FindArticle: regexp: %w", err)
-	}
-
 	for _, r := range rsss {
-		for _, it := range r.Channel.Items {
-			// very basic string matching on title to read an article
-			if regex.MatchString(strings.ToLower(it.Title)) {
-				item = it
-				break
-			}
+		item, err := r.FindArticle(substr)
```

Comment:
What was the reasoning for moving this logic?

Reply:
> I've been thinking about this a bit more and whilst I still want to add the `--feed` option for config, I don't see why this can't also be present as it is a very convenient way to view a single rss feed quickly.

Yep, even something like

```
nom url1 url2 url3
```

as a shortcut for

```
nom --feed=url1 --feed=url2 --feed=url3
```

Anyway, it's the just the interface, how the info gets in, udnrelying mechanics of how feeds are previewed should be the same.

Thank you for the code review! Very helpful. Let's see what I can do

Reply:
@guyfedwards initial implementation was a little different to what you see on the PR and it required being able to find an article in a single instance of `RSS`.

However, later on, I've changed the business logic so it'd "hook up" better into existing code yet decided to leave this helper method as generally useful. Happy to revert this change tho, - without the context of what my initial implementation was it indeed doesn't make sense :)

Reply:
Makes sense. I would revert it for this PR to keep the changes isolated.